go-app/templates/events.html

```diff
@@ -1,33 +1,162 @@
-
 <!DOCTYPE html>
 <html>
 <head>
-  <meta charset="utf-8" >
-  <title> Radar-log-parser</title>
- <link rel="stylesheet" href="/assets/styles.css">
- <style>
- </style>
+    <meta charset="utf-8" >
+    <title> Radar-log-parser</title>
+   <link rel="stylesheet" href="/assets/styles.css">
+   <style>
+     .details{
+        display: none;
+     }
+    
+     .banner_after,.banner_before{
+        width:100%;
+        padding:0.5%;
+        background-color: gray;
+        margin-bottom: 0.5%;
+     }
+     span{
+        cursor: pointer;
+        margin-left:10%;
+     }
+     #main{
+        margin-left:40%;
+     }
+    .event {
+        color: fuchsia;
+        cursor: pointer; 
+        text-align: left;
+        text-decoration: underline;
+     }
+  </style>
+  <script> 
+   function expandContent(element){
+     var content_elem = element.parentNode.nextElementSibling;
+     var expand_number = element.innerHTML;
+     if (expand_number == "Block"){
+        content_elem.style.display = "none";
+        return;
+     }
+     if (content_elem.style.display === "none") {
+        content_elem.style.display = "block";
+     }
+     var line_number = parseInt(content_elem.getAttribute("data-line-number"));
+     if(line_number<0 || line_number>=Math.pow(2, 52)){
+         return;
+     }
+     var size;
+     if(expand_number == "All"){
+        size =  Math.pow(2, 52) ;
+        if(element.parentNode.getAttribute("class")=="banner_before"){
```

Comment:
I'm confused why we only check "banner_before" and then make size = -size when expand_number = "All". Wouldn't we need to do this for every case?

Reply:
I did it because for banner_before, we are going up, so we are doing minus, and the number is decreasing but for banner_after it is increasing. Indeed, the line 49 is for banner_after